cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -44,6 +44,8 @@ message Survey {
     string name = 2;
     map<string, string> metadata = 3;
     ExternalId external_id = 4;
+    string crs = 5;
```

Comment:
Should this be the `CRS` type too?

Reply:
It seems like the `CRS` type is not used in v1, so I changed everything to use string. Not sure what's correct here.

@amorken?